.travis.yml

```diff
@@ -7,6 +7,14 @@ language: c
 
 compiler: [ gcc, clang ]
 
+cache:
+  ccache: true
```

Comment:
I don't know much about travis-ci but according to the documentation, it is defined as "cache: ccache"
https://docs.travis-ci.com/user/caching/

Reply:
Also there's the comment:
"to cache $HOME/.ccache and automatically add /usr/lib/ccache to your $PATH."

Reply:
That way of specifying cache is limited to ccache only

If we ever will want to add other caches, "cache: ccache" will not work

As for now we can use short form, you are right